tests/integration/integration_test.go

```diff
@@ -85,26 +86,26 @@ func TestEmptyGo(t *testing.T) {
 }
 
 // Tests emitting many engine events doesn't result in a performance problem.
-// func TestEngineEventPerf(t *testing.T) {
-// 	// Prior to pulumi/pulumi#2303, a preview or update would take ~40s.
-// 	// Since then, it should now be down to ~4s, with additional padding,
-// 	// since some travis machines (especially the OSX ones) seem quite slow
-// 	// to begin with.
-// 	benchmarkEnforcer := &assertPerfBenchmark{
-// 		T:                  t,
-// 		MaxPreviewDuration: 8 * time.Second,
-// 		MaxUpdateDuration:  8 * time.Second,
-// 	}
-
-// 	integration.ProgramTest(t, &integration.ProgramTestOptions{
-// 		Dir:          "ee_perf",
-// 		Dependencies: []string{"@pulumi/pulumi"},
-// 		Quick:        true,
-// 		ReportStats:  benchmarkEnforcer,
-// 		// Don't run in parallel since it is sensitive to system resources.
-// 		NoParallel: true,
-// 	})
-// }
+func TestEngineEventPerf(t *testing.T) {
```

Comment:
Good point. I assume it's still pretty good that we are run in-serial in the integration test package, but if we continue to see problems with flakiness and are sure it's related to concurrent resource usage, I can look into separating this out into its own test group. (e.g. the "no seriously, run in isolation because its super-sensitive to resources" test group.)